OnlineFlightSearchAPI/Models/AirportDetail.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineFlightSearchAPI.Models
+{
+    public class AirportDetail
+    {
+        public AirportDetail(string airportCode, string city, string country, DateTimeKind timeZone)
+        {
+            AirportCode = airportCode;
+            City = city;
+            Country = country;
+            TimeZone = timeZone;
+        }
+
+        public string AirportCode { get; set; }
+
+        public string City { get; set; }
+
+        public string Country { get; set; }
+
+        public DateTimeKind TimeZone { get; set; }
```

Comment:
Should probably use TimeZoneInfo. DateTimeKind just tells utc or local, is this what you are really using it for?
TimeZone is an in-built class, try avoiding conflicting names

Reply:
UTC DateTime is used to compare the specific location UTC Time